Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,591 @@
+      "use": "ec2_resolver",
+      "configs": {
+        "ec2_resolver": {
+          "MetadataBaseUrl": "http://169.254.169.254",
+          "AvailablePorts": [
+            {
+              "Min": 7777,
+              "Max": 7777
+            }
+          ]
+        },
+        "static_resolver": {
+          "Address": "server.local:8080"
+        }
+      }
+    },
+    "backend": {
+      "configs": {
+        "catena_backend": {
+          "url": "$CATENA_URL"
+        },
+        "noop_backend": {}
+      },
+      "use": "catena_backend"
+    },
+    "sidecar": {
+      "DebugPanelAddr": "localhost:8081",
+      "MatchManagementStrategy": {
+        "OverrideConnectionDetails": true,
+        "AutoPopulateIDs": true
+      }
+    }
+  }
+}
+```
+
+The key is to set the following fields:
+
+`app.backend.catena_backend.url` The url of your deployed backend.
+`app.resolver.configs.ec2_resolver.Min` - The min port to use for clients to connect.
+`app.resolver.configs.ec2_resolver.Max` - The max port to use for clients to connect.
+
+_For this demo, min and max port should be the same, and be the port your server runs on. If you are following this demo, it should be `7777`_
+
+> Also set is the `app.event_source.use` field, which is `catena_api_event_source.` If your server implements the [Catena Server Manager](https://github.com/CatenaTools/catena-tools-core/blob/main/Protos/api/v1/catena_server_manager.proto) interface
+> it can make requests to this backend.
+
+After which you can launch the sidecar using `cmd.exe`
+
+<procedure collapsible="true" title="Run the sidecar">
+<step>
+Press <shortcut>Ctrl+R</shortcut> or Start -> Run
+</step>
+<step>Type cmd.exe and click run</step>
+<step>Change to the directory you are running the sidecar from <code>cd C:\Path\To\sidecar-windows-amd64.exe</code></step>
+<step>Launch the sidecar <code>sidecar-windows-amd64.exe</code></step>
+</procedure>
+
+At this point you should see the sidecar start up. It will resolve connection details and make requests on behalf of the game server. Our lyra demo is setup to request matches through the sidecar.
+
+> If you don't want to interface with the sidecar or want the quickest path to a working setup, the sidecar can pretend to be a server and request matches on behalf of your server. To do this, simply change `app.event_source.use` to `virtual_server_event_source`. This tells the sidecar to act as a server and request new matches
+> every 5 seconds. So long as your server is listening on port 7777 clients will be able to connect to your server.
+> 
+> If you restart the sidecar with this config, you will see it requesting matches repeatedly.
+> 
+> See [Match Broker](Match-Broker.md) to read about how Catena manages servers.
+
+
+## Testing it out
+
+Next we will download the sample game and test it out. In this case we will use a prebuilt version of the catena lyra demo. It can be found [here](https://catena-public-content.s3.amazonaws.com/WindowsClient.zip).
```

Comment:
catena lyra demo -> Catena Lyra Demo

(The whole tutorial should get a pass on capitalization. Not the end of the world, but is good for correctness)